packages/terra-form-input/src/Input.scss

```diff
@@ -0,0 +1,92 @@
+:local {
+  .form-input {
```

Comment:
The input appears much taller than before, which is ok, but is also appears taller compared to buttons. Buttons are 30px tall and the input is 35px tall. We'll want the input and button to have the same computed height for the default theme. This helps visually balance the input when its next to a button in an inline form. I can pair with you and figure out what all changes are needed, usually its a mix of line-height and padding.

Reply:
I set something up for Monday